pages/api/v1/list-employees.ts

```diff
@@ -0,0 +1,32 @@
+import { prismaClient } from "../../../lib/prisma-client";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (typeof req.query.userId !== "string") {
```

Comment:
Why checking for string instead of checking for null or undefined?

Reply:
@johnmosesman 

Theres a type coming off of NextApiRequest for query where string[] is a possibility. 

I had the conditional with a bunch of OR, but ended up using ☝️ 

![CleanShot 2023-07-28 at 12 35 56](https://github.com/FlatFilers/HCMShow/assets/12041898/bf93e0ec-28f7-44f1-9732-3763264c7f3d)

Reply:
Edited. Made it a bit more compact